keyboards/yushakobo/primer29/rules.mk

```diff
@@ -0,0 +1,4 @@
+# This file intentionally left blank
+CUSTOM_MATRIX = lite
+
```

Comment:
```suggestion
```

Reply:
This change makes it impossible to build, so it is being returned.